weavertest/internal/logging/logger_test.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2023 Google LLC
```

Comment:
I think weavertest/internal/simple/simple_test.go should also show the issues? If so maybe we don't need this new directory?